ArduCopter/RC_Channel.cpp

```diff
@@ -180,7 +180,7 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
             break;
 
         case AUX_FUNC::SAVE_TRIM:
-            if ((ch_flag == HIGH) && (copter.control_mode <= Mode::Number::ACRO) && (copter.channel_throttle->get_control_in() == 0)) {
+            if ((ch_flag == HIGH) && (copter.control_mode <= &copter.mode_acro) && (copter.channel_throttle->get_control_in() == 0)) {
```

Comment:
pointer comparison? needs to be a method on the mode

Reply:
We should replace, "(copter.control_mode <= &copter.mode_acro)" with a new member on the Mode class.  So we should do this:

1. add a new method to the Mode class called something like, "bool autotune_supporte() const { return false; }"
2. acro and stabilize modes should implement this new method but return true.
3. use this new method in the case AUX_FUNC::SAVE_TIME::

Reply:
Thank you for the review.
I fixed it using allows_xxx().